.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2.1
+
+jobs:
+  release-patch-with-changelog:
+    machine:
+      image: ubuntu-1604:201903-01
+      docker_layer_caching: true
+
+    steps:
+      - checkout
+
+      - add_ssh_keys:
+          fingerprints:
+            - "f5:ca:b0:c5:06:26:aa:ec:c8:a7:75:57:f0:e7:c2:f3"
+
+      - run: pwd
+
+      - run: ls -ltra
+
+      - run: git branch
+
+      - run:
+          name: release-patch-with-changelog-circleci
+          command: |
+            if git status | grep 'nothing to commit, working tree clean'; then
+              echo "==============================================================================================="
+              echo "release-patch-with-changelog-circleci"
+              echo "==============================================================================================="
+              git config --global user.email "info@binbash.com.ar"
```

Comment:
@exequielrafaela I think you can set env vars in your repositories -- not sure if they are set per repository or if they can be made globally available). That way you store email and name there and then use them in your pipelines to improve that aspect just a little bit.

Reply:
@diego-ojeda-binbash Thanks for taking the time for the review and great peace of advice. Do you mean setting ENV VARS in the  in the `./circleci/config.yml`? 

Something like: 
```
version: 2
jobs:
  build:
    docker:
      - image: smaant/lein-flyway:2.7.1-4.0.3
      - image: circleci/postgres:9.6-jessie
      # environment variables for all commands executed in the primary container
        environment:
          POSTGRES_USER: conductor
          POSTGRES_DB: conductor_test
```
source: https://circleci.com/docs/2.0/env-vars/

Reply:
@diego-ojeda-binbash OJ I follow you now! Great point, this is actually possible per project (https://circleci.com/blog/devops-did-not-exist/) Sadly it's not cross-org, but at least every job that will be triggered inside the same project (1 project x repo) can use this ENV Vars from this CircleCI config. For example each 3 jobs for this same repo project will have centralized access to them:
1. **job:** sec-scan
2. **job:** build-test 
3. **job:** release

The main **con** is that is a manual UI config per project :-1: . Not sure if it's what we where expecting. However a big **pro** is that it works like a vault, keeping this ENV Vars protected from being access in plain text from the ./circleci/config.yml

![image](https://user-images.githubusercontent.com/18539704/64663109-1b631b80-d421-11e9-9f40-e7fbb713de26.png)


What are your final thoughts about it?

Reply:
@exequielrafaela Sounds like a step in the right direction. 👍

Reply:
@diego-ojeda-binbash Great, I'll update the PR having the CircleCI env vars configured and validated before merging! Let's see if everything works as expected after the PR is actually merged :muscle: